cyclic_boosting/features.py

```diff
@@ -266,6 +269,11 @@ def prepare_feature(self) -> None:
         if self.smoother is not None:
             self.smoother.smoothed_y_ = self.factors_link[:-1].copy()
 
+    def set_feature_bin_weighted_average(self, neutral_factor_link: float) -> None:
```

Comment:
I would use another name for this function, e.g., set_feature_bin_deviations_from_neutral.